### PR TITLE
Feature/enabled platforms

### PR DESCRIPTION
**Pull request by alaouy**
Discussed here #13

### CODE REVIEW

**Comment by ain** [comment]
Great job @alaouy! Thanks a lot for the PR. I'll push the review comments shortly, minor stuff.

File: test/spec/smartbanner_spec.js [hunk 1]
```diff
@@ -224,6 +225,76 @@ describe('SmartBanner', function() {
...
+          <meta name="smartbanner:author" content="SmartBanner Contributors">
+          <meta name="smartbanner:price" content="FREE">
+          <meta name="smartbanner:price-suffix-apple" content=" - On the App Store">
+          <meta name="smartbanner:price-suffix-google" content=" - In Google Play">
+          <meta name="smartbanner:icon-apple" content="icon--apple.jpg">
+          <meta name="smartbanner:icon-google" content="icon--google.jpg">
+          <meta name="smartbanner:button" content="View">
+          <meta name="smartbanner:button-url-apple" content="https://itunes.apple.com/us/genre/ios/id36?mt=8">
+          <meta name="smartbanner:button-url-google" content="https://play.google.com/store">
+          <meta name="smartbanner:enabledPlatforms" content="android">
```
Comment (ain) [inline test/spec/smartbanner_spec.js]: Let's keep to `enabled-platforms` here just for the sake of consistency.

File: test/spec/smartbanner_spec.js [hunk 2]
```diff
@@ -224,6 +225,76 @@ describe('SmartBanner', function() {
...
+          <meta name="smartbanner:author" content="SmartBanner Contributors">
+          <meta name="smartbanner:price" content="FREE">
+          <meta name="smartbanner:price-suffix-apple" content=" - On the App Store">
+          <meta name="smartbanner:price-suffix-google" content=" - In Google Play">
+          <meta name="smartbanner:icon-apple" content="icon--apple.jpg">
+          <meta name="smartbanner:icon-google" content="icon--google.jpg">
+          <meta name="smartbanner:button" content="View">
+          <meta name="smartbanner:button-url-apple" content="https://itunes.apple.com/us/genre/ios/id36?mt=8">
+          <meta name="smartbanner:button-url-google" content="https://play.google.com/store">
+          <meta name="smartbanner:enabledPlatforms" content="ios">
```
Comment (ain) [inline test/spec/smartbanner_spec.js]: Let's keep to `enabled-platforms` here just for the sake of consistency.

File: test/spec/smartbanner_spec.js [hunk 3]
```diff
@@ -224,6 +225,76 @@ describe('SmartBanner', function() {
 
     });
 
+    context('when enabledPlatform set to android, but is openned on a iPhone', function() {
```
Comment (ain) [inline test/spec/smartbanner_spec.js]: Let's improve here a bit, e.g. `when enabled-platform set to android, but opened on iOS`

File: test/spec/smartbanner_spec.js [hunk 4]
```diff
@@ -224,6 +225,76 @@ describe('SmartBanner', function() {
...
+        smartbanner.publish();
+      });
+
+      it('expected to not add anything to body', function() {
+        expect(document.querySelector('.js_smartbanner')).to.be.null;
+      });
+
+    });
+
+    context('when enabledPlatform set to ios, but is openned on a Android', function() {
```
Comment (ain) [inline test/spec/smartbanner_spec.js]: Let's improve here a bit, e.g. `when enabled-platform set to ios, but opened on Android`

File: test/spec/smartbanner_spec.js (hunk 2 above)
Comment (alaouy) [inline test/spec/smartbanner_spec.js]: Oups, that was a typo :/

**Comment by alaouy** [comment]
@ain I updated the test file and included this feature in the build.

**Pull request merged**